internal/credentials/spiffe_test.go

```diff
@@ -178,3 +181,25 @@ func (s) TestSPIFFEIDFromState(t *testing.T) {
 		})
 	}
 }
+
+func (s) TestSPIFFEIDFromCert(t *testing.T) {
+	data, err := ioutil.ReadFile(testdata.Path("x509/spiffe_cert.pem"))
+	if err != nil {
+		t.Fatalf("Failed to load the test credentials: %v", err)
```

Comment:
Please identify the function and its inputs in the error message. Otherwise there is no way of knowing which file read failed. (this comment applies to all other test error messages as well).

Something like this would be more appropriate:
`t.Fatalf("ioutil.ReadFile(%s) failed: %v", path, err)`